docs/developer-guide/working-with-rules.md

```diff
@@ -62,6 +62,7 @@ The source file for a rule exports an object with the following properties.
     * `category` (string) specifies the heading under which the rule is listed in the [rules index](../rules/)
     * `recommended` (boolean) is whether the `"extends": "eslint:recommended"` property in a [configuration file](../user-guide/configuring.md#extending-configuration-files) enables the rule
     * `url` (string) specifies the URL at which the full documentation can be accessed
+    * `suggestion` (boolean) specifies whether rules can return suggestions
```

Comment:
In the Suggestions RFC it said:

> Update `docs` section of rule metadata to include `suggestion: true|false` to make it easier to identify rules that can return suggestions.

Did I understand this correctly here? If so, should I go and add this field to all the rules in the repo to identify they can't return suggestions? I notice there isn't a similar `docs` field for fixes. How is this actually going to be used and be useful?

Reply:
I think it's fine to assume that if `suggestion` keyword isn't present that means `false`. That's what we do with `fix`. So no need to update existing rules.

Reply:
Okay. Should I omit this from the docs then?

Reply:
Ah, got it